### PR TITLE
Rubber fix + New E2/SF functions

### PR DESCRIPTION
**Pull request by KemGus**
-Fixed rubber taking heat effectiveness into account when dealing with spall

-Added Function to see whether different crew seats are linked or not
-Added Function to get heat from a specified ace entity
Function to get ACE Version (git and current)

### CODE REVIEW

**Comment by KemGus** [comment]
Note: coudn't get acfGetCrew function to work in E2. Works in SF though. No idea why it no worky

File: lua/starfall/libs_sv/acf.lua [hunk 1]
```diff
@@ -2227,6 +2245,51 @@ function ents_methods:acfDragCoef()
...
+
+	if not (this and this:IsValid()) then
+		SF.Throw("Entity is not valid", 2)
+	end
+
+	if restrictInfo( this ) then
+		return {}
+	end
+
+	local Crew = this.CrewLink
```
Comment (CheezusChrust) [inline lua/starfall/libs_sv/acf.lua:2286]: This should be `local Crew = table.Copy(this.CrewLink)` or bad things will happen

File: lua/starfall/libs_sv/acf.lua [hunk 2]
```diff
@@ -2227,6 +2245,51 @@ function ents_methods:acfDragCoef()
...
+
+	return Crew
```
Comment (CheezusChrust) [inline lua/starfall/libs_sv/acf.lua:2288]: This needs to be changed to use table.Copy

**Review by CheezusChrust — CHANGES_REQUESTED** [review]

File: lua/starfall/libs_sv/acf.lua (hunk 2 above)
Comment (KemGus) [reply lua/starfall/libs_sv/acf.lua:2288]: fixed

**Review by CheezusChrust — APPROVED** [review]

**Pull request merged**